README.md

```diff
@@ -565,6 +566,30 @@ $ /system/bin/tee-helloworld
 ```
 Enjoying OP-TEE on A80 board.
 
+---
+### 4.8 HiKey board
```

Comment:
Either we keep 4.8 here since MediaTek PR are using 4.7 or we have a race between commits.